tasks/setup_Debian.yml

```diff
@@ -25,10 +25,9 @@
     executable: /bin/bash
 
 - name: Add the Okta ASA repo
-  ansible.builtin.copy:
-    content: >-
+  ansible.builtin.apt_repository:
```

Comment:
Ideally we would use [`ansible.builtin.deb822_repository`](https://docs.ansible.com/ansible/latest/collections/ansible/builtin/deb822_repository_module.html) where possible.  See, for example, [here](https://github.com/cisagov/ansible-role-docker/blob/a3abb6750151e52109cb06afad1990f08e541597/tasks/setup_Debian.yml#L20-L56).

Reply:
I was planning to add that functionality in a follow-up PR.

Reply:
I'm fine with this happening in a follow-on PR.

The nice thing about the example I provided is that it only checks for the presence of Buster.  That means the logic will still work as written as new Debian versions are released.  Or is there another reason to prefer the logic in the backports role?

Reply:
No reason to prefer it, I would just prefer we be consistent. I knew backports role did a similar split-config with some getting DEB822 and some getting classic which is why I was using it as a reference. I was more just mentioning that my original intent was to use that logic and not arguing in favor of that logic.